client/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,41 @@
+import { FaInstagram, FaLinkedin, FaTiktok } from "react-icons/fa";
+import styles from "./Footer.module.css";
+
+export default function Footer() {
```

Comment:
There's some inconsistent tabbing, I believe we're sticking with 2 tab space instead of 4

Reply:
Fixed